### ISSUE TITLE
Inaccessibility : The titles of the posts' highlight are custom links which do not receive keyboard focus.

### ISSUE THREAD

**Issue by peternjugunamwaura** [a11yproject/a11yproject.com#1532]
Thank you for reporting a bug. Please fill out the following information:

## Bug description

The title of the posts below the heading "Learn the fundamentals and principles behind accessible design" on the home page do not receive keyboard focus. Examples of these titles include:- "Testing Accessibility" and "Introduction to Web Accessibility". The titles act as links that lead to the individual posts.

## Expected outcome

Ensure that all interactive elements receive keyboard focus. For custom elements such as these titles, add attributes `tabindex="0"` and `role="link"`.

## Steps to reproduce

 1. Open https://www.a11yproject.com/ on Google Chrome.
 2. Press TAB key to navigate to the link "Explore Resources".
 3. Using the TAB key verify if the next link which is "Testing Accessibility" receives keyboard focus.

## Technical info
The code reference for the element with the bug is `<h3 class="u-font-size-body-medium c-homepage-card__title" onclick="location.href='/resources/testing-accessibility/'">Testing Accessibility</h3>`. Since this heading has an onclick listrener that changes the URL, it should have a role of a link. Furthermore, to ensure that it receives keyboard focus, it should an attribute tabindex=0. The compliant code example would therefore be:- `<h3 tabindex="0" role="link" class="u-font-size-body-medium c-homepage-card__title" onclick="location.href='/resources/testing-accessibility/'">Testing Accessibility</h3>`
###  What device were you using when you encountered the bug?

Laptop

### What operating system were you using? What version is the operating system?

Windows 10

### What web browser were you using? What version is the web browser?

Google Chrome 111.0.5563.65

### Assistive technology and version (optional)

No AT.

## Additional information

![Screenshot_30](https://user-images.githubusercontent.com/61505930/226399700-5dfbde3b-10ef-46c1-b795-0ff017d334db.png)

## Terms

[ ] I have read, and agree to follow this project's [Code of Conduct](https://www.a11yproject.com/code-of-conduct/)

**Comment by ericwbailey** [comment]
This is an intentional design. The other links provide an accessible name for the link, and the `onClick` is done to remove redundant links from tabbing, rotor, and other forms of interactive content navigation.

**Comment by Peter-Peter254** [comment]
Thanks. will do
